### ISSUE TITLE
Theme "Windows10 (WIP)" not working

### ISSUE THREAD

**Issue by HJMeyer** [ramensoftware/windows-11-start-menu-styling-guide#15]
After installing Theme "Windows10 (WIP)" with the Windhawk Windows 11 Start Menu Styler mod, I was pleasantly surprised at first. But then it got strange:

1st problem: Alternating between new and original start menu:
After successfully installing the “Windows10 (WIP)” theme from the Windhawk website, the old menu opens from time to time, but the entries are not clickable. 

2nd problem: Start/restart menu not clickable:
Then I tried to restart and clicked on the On/Off button in the new menu. The small menu with the options Log in, Save energy, Shut down and Restart opens, but it is not clickable.

3rd problem: New menu disappears after second restart:
I then managed to restart without using the menu. After the first restart, the new menu opened properly and  and worked as expected. I tested the restart. It also worked, but after this second reboot the old menu appeared (and worked). The newly installed theme “Windows10 (WIP)” never appeared again, even after several reboots. What has happened?

Thank you, HM

Windows 11 Pro 23H2
AMD Ryzen 9 7900 12-Core Processor, 3.70 GHz
Windows 11 start menu styling
Theme: Windows10 (WIP)

**Comment by m417z** [comment]
@bbmaster123 FYI

**Comment by HJMeyer** [comment]
Yes, exactly: [That ](https://github.com/bbmaster123) is the webpage where I found this mod and installed it following the instructions under [Installation](https://github.com/bbmaster123/windows-11-start-menu-styling-guide/blob/main/Themes/Windows10/README.md#installation).
What information did I miss?

**Comment by bbmaster123** [comment]
@m417z thanks for the heads up! I probably would have missed this...

@HJMeyer when did you copy the code? I just updated it yesterday, it should fix most issues. You can tell if its the latest if your username text is gone from the left side, I've removed that.
Let me know, as I have this running on 2 systems right now, and all buttons are clickable for me. Plus I'm eager to squash any remaining major issues.

Also maybe you can include your windows build and any errors or warnings showing in the windhawk log? You can find that under start menu styler > advanced > set detailed logs from dropdown > show output log

here's how it looks on my end:
https://github.com/ramensoftware/windows-11-start-menu-styling-guide/assets/92948468/7dba9c37-9f8e-4851-a960-a5c2401888ec
